tests/test_pycrefine.py

```
@@ -1368,16 +1367,16 @@ def test_while_loop_uses_jump_absolute_back_edge(self):
            12  JUMP_ABSOLUTE 0        <- back-edge
            14  RETURN_VALUE           <- loop exit (is_jump_target=True)
         """
-        Instr = BytecodeInstruction
+        I = BytecodeInstruction
```

Ruff (0.15.6)

[error] Ambiguous variable name: `I`

(E741)